ceph.spec.in

```diff
@@ -693,6 +693,7 @@ Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:	lvm2
 Requires:	sudo
 Requires:	libstoragemgmt
+Requires:	python%{python3_pkgversion}-ceph-common = %{_epoch_prefix}%{version}-%{release}
```

Comment:
note py-ceph-common is no longer py2 compatible. At least for master and octopus. 

Also it is not yet backported to nautilus.

Reply:
Yep am aware. This is a follow up for the `drive-groups` subcommand and will only be backported to octopus.